.github/julia/generate_binaries.jl

```diff
@@ -47,6 +30,12 @@ for (platform, libdir, ext) in platforms
       # Unzip the tarball of the dependencies
       run(`tar -xzf products/$platform/deps.tar.gz -C products/$platform`)
 
+      # Copy the license of each dependency
+      for folder in readdir("products/$platform/deps/licenses")
+        cp("products/$platform/deps/licenses/$folder", "products/$platform/share/licenses/$folder")
```

Comment:
Do we need
`mkdir "products/$platform/share`
`mkdir "products/$platform/share/licenses`
here?